js/logo.js

```diff
@@ -5251,7 +5251,7 @@ function Logo() {
             if (childFlow != null) {
                 var queueBlock = new Queue(childFlow, 1, loopBlkIdx);
                 // We need to keep track of the parent block to the
-                // child flow so we can unlightlight the parent block
+                // child flow so we can unhighlight the parent block
```

Comment:
You need to do a rebase so you don't drag in all these old changes.

Reply:
No,the last typo was in  line 1866 ,this typo is same as that one but in line 5254